build.wake

```diff
@@ -117,10 +116,9 @@ global def runRocketChipGenerator options =
       Some name = name, Nil
       None = Nil
     which "java", "-cp", classpath, main,
-    relTargetDir,
-    rootPackage, topModule,
-    rootPackage, configs,
-    baseFileName
+    "-td", relTargetDir,
+    "-T", topModule,
+    "-C", configs, Nil
```

Comment:
Unless if I'm missing something, I don't think this is a correct translation between the old CLI and the new CLI. This looks like it is throwing out the `rootPackage` argument, which I assume was the package containing the `topModule`, but we're not passing it to the command line interface anymore. Same goes for the configs.

Also, it looks like the `baseFileName` has been dropped altogether, which makes determining the actual base file name more difficult, as you can see in the `def topModPackage` statement below. See https://github.com/chipsalliance/rocket-chip/pull/2350 for how I had attempted to do this prior to the stages API being merged in, but I think it results in an easier way to handle the base name, since we set it on the Wake side rather than trying to derive it from what we think the rocket-chip Generator is trying to do.

I'd suggest that we make the following changes:

1. Construct the full package paths for the top module and config before passing them in to the rocket-chip generator CLI.
2. Continue passing in the `baseFileName` argument
3. Pull in pieces of my changes in https://github.com/chipsalliance/rocket-chip/pull/2350 to compute the derived `outputBaseName`. Feel free to ignore the DTSJSON thing for now.
4. (Nit:) Using the fully-spelled out forms of the command line flags, since they make it easier for someone who is not familiar with the CLI to read. `--top-module` is much more understandable than `-T`, IMO.

Reply:
I think the current layout kind of makes it self-explanatory i.e. td -> target dir, t -> topmodule, c -> configs and looking at code --name for basefilename

Reply:
I tested these changes with a bumped api-scala-sifive(https://github.com/sifive/api-scala-sifive/tree/v0.1.1) where it seemed like it didn't require _root_